src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -324,6 +324,11 @@ private static bool TryGetTextForCombinationKeyword(SyntaxToken token, ISyntaxFa
                 case SyntaxKind.InternalKeyword when ModifiersContains(token, syntaxFacts, SyntaxKind.ProtectedKeyword):
                     text = "protectedinternal_CSharpKeyword";
                     return true;
+
+                case SyntaxKind.UsingKeyword when token.GetNextToken().IsKind(SyntaxKind.StaticKeyword):
+                case SyntaxKind.StaticKeyword when token.GetPreviousToken().IsKind(SyntaxKind.UsingKeyword):
+                    text = "usingstatic_CSharpKeyword";
```

Comment:
You have a hyphen between "using" and "static" in the docs PR. Not really fussed which one you change, but they need to match, so I'm blocking this for now.

Reply:
Yes, I just saw that as well.  I decided to switch to the hyphen last night (too late in the evening) and missed changing it there.  I saw the test failure this morning and just pushed the missing change.